policy/modules/services/aptcacher.te

```diff
@@ -5,6 +5,10 @@ policy_module(aptcacher, 1.0.0)
 # Declarations
 #
 
+type acngtool_t;
+type acngtool_exec_t;
+application_domain(acngtool_t, acngtool_exec_t)
```

Comment:
You indicate that this is a application, yet you are missing role access (and various interfaces):

```
attribute_role acngtool_roles;
role acngtool_roles types acngtool_t;
```

```
########################################
## <summary>
##	Execute acngtool in the acngtool domain.
## </summary>
## <param name="domain">
##	<summary>
##	Domain allowed to transition.
##	</summary>
## </param>
#
interface(`aptcacher_domtrans_acngtool',`
	gen_require(`
		type acngtool_t, acngtool_exec_t;
	')

	corecmd_search_bin($1)
	domtrans_pattern($1, acngtool_exec_t, acngtool_t)
')

########################################
## <summary>
##	Execute acngtool in the acngtool domain, and
##	allow the specified role the acngtool domain.
## </summary>
## <param name="domain">
##	<summary>
##	Domain allowed to transition.
##	</summary>
## </param>
## <param name="role">
##	<summary>
##	Role allowed access.
##	</summary>
## </param>
#
interface(`aptcacher_run_acngtool',`
	gen_require(`
		attribute_role acngtool_roles;
	')

	aptcacher_domtrans_acngtool($1)
	roleattribute $2 acngtool_roles;
')
```